src/lib.rs

```diff
@@ -40,6 +40,15 @@
 #![doc(html_root_url = "https://docs.rs/cloudevents-sdk/0.3.1")]
 #![deny(broken_intra_doc_links)]
 
+#[cfg(feature = "ce-actix")]
```

Comment:
Just name the feature `actix` without prepending with ce? Same for the below modules

Reply:
That was my initial approach. As you can see from my [2nd commit](https://github.com/cloudevents/sdk-rust/pull/131/commits/14ac80ebfb09bd5cff527d0404ba3b5551a2029d), I had a conflict with `reqwest`. I'm happy to resolve that however you like, but I opted for a consistent prefix.